content/releases/daito/_index.md

```diff
@@ -0,0 +1,156 @@
+
+## Release Information
+### Opensds Changes
+
+- Hotpot API updates for basic data provisioning
+- Hotpot API additional parameters added
+- Swagger API validation added
+
+### What's new
+
+- Host Management APIs initial version integrated
+- updated to openapi 3.0
+- New South Bound Driver added (Beta)
+- NetApp ONTAP Select
+- IBM Spectrum Scale
+
+### NBP Changes
+
+- CSI Block and File Separation
+- Hostname and Host attachment changes for CSI
+- Bug fixes
+
+### What's new
+
+- VMWare support (Initial support for NGC and vRO)
+- Hostname and Host attachments for NGC and vRO
+
+### Known issues
+
+- FlexVol is not supported in this version
+
+## Multicloud Changes
+
+- Integration with YIG to enhance multicloud
+- Architecture and Design updates to handle YIG integration and optimization
+- Bug fixes and performance improvements
+
+### What's new
+
+- Support for YIG Storage backend as on-premise cloud
+- Copy object
+- ACL (custom) for Buckets [public-read, public-read-write and private (private is the default)]
+- ACL (custom) for Objects [public-read and private (private is the default)]
+- Multipart upload and download for non encrypted buckets (except Ceph and GCP backends)
+- SSE with Server Managed Keys (put and get using only normal upload(non-multipart) and download. Please use API interface)
+
+### Known issues
+
+- Bucket lifecycle is not fully verified for Ceph and GCS
+
+## Opensds Dashboard Changes
+
+_The dashboard is mainly intended to showcase the features of opensds. It is only for demo purpose currently.
+[For the complete feature and advanced capabilities of opensds please refer/use backend API /documentations]_
+
+- profile page updates for API changes
+
+## What's new
+
+- Host Management support
+- Volume attach/detach support in OpenSDS UI
+- Multicloud feature support for SSE, ACL, Copy/Paste Object
+
+## Opensds-installer Changes
+
+- Installer script optimized for better install experience
+
+## Bug fixes
+- Modular and more controlled installation script for different projects
+- CSI changes for Block and File Separation
+
+## What's new
+
+- Multicloud installation to support YIG integration and YIG Storage Backend
+- Install support for NetApp ONTAP select driver
+
+**Note:** For VMWare NBP installations, please refer to VMWare NBP Documentation. It is not part of installer
+
+## Documentation Changes
+
+- Redesign and updates on the documentation
+
+## What's new
+
+- New repo for documentation and migration
+
+## Orchestration Changes
+
+- Changes in the Volume Provisioning action for host attachment
+
+## Anomaly Detection
+
+- No Changes
+
+## How to use the Release:  
+
+### From Release Binaries of OpenSDS Installer
+
+Steps:
+
+```
+#Download the installer binaries from https://github.com/opensds/opensds-installer/releases/tag/v0.10.0 (Use wget or direct download using browser)
```

Comment:
Can you please make it bulleted points for all the installation steps?

Reply:
Fixed.